### PR TITLE
Issue #5200: Shorthand [] for array() breaking php5.3 tests in GHA

### LINKED ISSUE #5200: Fixed: Shorthand [] for array() breaking php5.3 tests in GHA
I recently noticed this error in a PR, for php5.3:

```
Run core/scripts/install.sh --db-url=***127.0.0.1/backdrop
PHP Parse error:  syntax error, unexpected '[' in
/home/runner/work/backdrop/backdrop/core/modules/filter/filter.install on line 541
Error: Process completed with exit code 255.
```

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/5200

### CODE REVIEW

**Comment by klonos** [comment]
Got `"Failed to get available update data for one project." found only once` in one of the php5.3 tests. Restarting...

**Comment by bugfolder** [comment]
Looks good to me.

File: core/modules/filter/filter.install [hunk 1]
```diff
@@ -538,8 +538,8 @@ function filter_update_1004() {
         'multiple' => FALSE,
         'remember' => 0,
         'default_group' => 'All',
-        'default_group_multiple' => [],
-        'group_items' => []
+        'default_group_multiple' => array(),
+        'group_items' => array()
```
Comment (ghost) [inline core/modules/filter/filter.install]: Can you add a trailing comma @klonos?

```suggestion
        'group_items' => array(),
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: core/modules/filter/filter.install (hunk 1 above)
Comment (klonos) [reply core/modules/filter/filter.install]: Good catch @BWPanda 👍🏼  ...that code must have been copy/pasted almost verbatim from a .json exported file, so both those `[]` as well as this missing comma after the last array element are a result of that.

**Comment by klonos** [comment]
Rerunning tests.

**Comment by klonos** [comment]
...got this random failure this time: https://github.com/backdrop/backdrop-issues/issues/5201

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Merged into 1.x.

**Pull request merged**